tools/validate_configs/test_configs/apt-collision.yaml

```diff
@@ -39,17 +39,12 @@ deployment_groups:
     kind: terraform
     id: network1
 
-  - source: ./modules/scripts/startup-script
```

Comment:
We likely want to leave at least one local reference in our `test_configs` since this is still a valid use we want coverage over.